### PR TITLE
Add presubmits for kueue 0.6

### PR DESCRIPTION
**Pull request by alculquicondor**
Part of https://github.com/kubernetes-sigs/kueue/issues/1729

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-6.yaml [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+        - name: GOMAXPROCS
+          value: "4"
+        resources:
+          requests:
+            cpu: "4"
+            memory: "6Gi"
+          limits:
+            cpu: "4"
+            memory: "6Gi"
+  - name: pull-kueue-test-e2e-release-0-6-1-25
```
Comment (kannon92) [inline config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-6.yaml:61]: I think we should not add 1.25 tests

Comment (alculquicondor) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-6.yaml]: You are right, we are not testing this for the main branch already.

**Comment by kannon92** [comment]
LGTM.

Not sure if you want approval from kueue maintainer so I'll leave the label for @tenzen-y

**Comment by alculquicondor** [comment]
Please do LGTM :)

**Pull request merged**